docs/outlook/smart-alerts-onmessagesend-walkthrough.md

```diff
@@ -13,15 +13,15 @@ The `OnMessageSend` and `OnAppointmentSend` events take advantage of Smart Alert
 The following walkthrough uses the `OnMessageSend` event. By the end of this walkthrough, you'll have an add-in that runs whenever a message is being sent and checks if the user forgot to add a document or picture they mentioned in their email.
 
 > [!IMPORTANT]
-> The `OnMessageSend` and `OnAppointmentSend` events are only available in preview with a Microsoft 365 subscription in Outlook on Windows. For more details, see [How to preview](autolaunch.md#how-to-preview). Preview events shouldn't be used in production add-ins.
+> The `OnMessageSend` and `OnAppointmentSend` events are only available in preview with a Microsoft 365 subscription in Outlook on Windows and on the web. For more details, see [How to preview](autolaunch.md#how-to-preview). Preview events shouldn't be used in production add-ins.
 
 ## Prerequisites
 
 The `OnMessageSend` event is available through the event-based activation feature. To understand how to configure your add-in to use this feature, use other available events, configure preview for this event, debug your add-in, and more, refer to [Configure your Outlook add-in for event-based activation](autolaunch.md).
 
 ## Set up your environment
 
-Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator), which creates an add-in project with the Yeoman generator for Office Add-ins.
+Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator), which creates an add-in project with the [Yeoman Generator for Office Add-ins](../develop/yeoman-generator-overview.md).
```

Comment:
The name is actually "Yeoman generator for Office Add-ins" at https://github.com/OfficeDev/generator-office. Can you check with Ricky - perhaps we need to update that README?
```suggestion
Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator), which creates an add-in project with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
```

Reply:
> The name is actually "Yeoman generator for Office Add-ins" at https://github.com/OfficeDev/generator-office. Can you check with Ricky - perhaps we need to update that README?

@Rick-Kirkham should we use 'Yeoman generator for Office Add-ins' per https://github.com/OfficeDev/generator-office or 'Yeomen Generator for Office Add-ins' per https://docs.microsoft.com/en-us/office/dev/add-ins/develop/yeoman-generator-overview?

Reply:
Following up offline. Will address this in a separate PR, if needed, so that this article can be published today.